cmd/helm/search/search_test.go

```diff
@@ -196,7 +196,7 @@ func TestSearchByName(t *testing.T) {
 		},
 		{
 			name:  "description search, two results",
-			query: "two",
```

Comment:
These test cases still need to pass, please add uppercase test cases as additional cases.